config.yml

```diff
@@ -3,8 +3,8 @@ AWS_BUCKET_NAME: "forestforesight-public"
 AWS_BUCKET_REGION: "eu-west-1"
 EARLIEST_DATA_DATE: "2021-01-01"
 DEFAULT_GROUNDTRUTH: "groundtruth6m"
-CODE_LOCATION: "D:/WWF/Repos/fork-1/ForestForesight"
+DATA_FOLDER: "C:/Kodingan3/FFdatas"
```

Comment:
can we also keep this folder empty when merging to develop, and maybe a comment that this should be empty string here